app/javascript/pulfalight/toc.es6

```diff
@@ -43,8 +45,10 @@ export default class TocBuilder {
     this.element.on('ready.jstree', (e, data) => {
       const selectedId = this.element.jstree().get_selected()[0]
       const selectedElement = $(`#${selectedId}`)
-      const scrollOffset = selectedElement.offset().top - selectedElement.offsetParent().offset().top - 60
-      this.element.scrollTop(scrollOffset)
+      if (selectedElement.length > 0) {
```

Comment:
Suggestion: Do you know what this is for exactly? I was pretty confused by it, a comment might help if you know what's going on?